accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -325,6 +354,30 @@ not a subtype of the class type associated with the class in which it is defined
 (specifically, it is an error to return a nullable type from a factory
 constructor for any class other than `Null`).
 
+It is an error if any case of a switch statement except the last case (the
+default case if present) **may complete normally**.  The previous syntactic
+restriction requiring the last statement of each case to be one of a delimited
+set of statements (break, continue, return, throw, or rethrow) is removed.
+
+Given a switch statement which switches over an expression `e` of type `T`,
+where the cases are dispatched based on expressions `e0...ek`:
+  - It is no longer required that the `ei` be instances of the same class.
```

Comment:
-> ... that the values of the `ei` expressions are instances  ...

The ei are expressions, not values, and only values can be instances of classes.

Reply:
done.